cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgisSink.java

```diff
@@ -549,16 +549,18 @@ public void aggregate(NGSIEvent cygnusEvent) {
                 } else {
                     // create part of the column with the current attribute (a.k.a. a column)
                     if (attrNativeTypes) {
-                        if (attrType.equals("Number")) {
-                            column += "," + attrValue + ",'"  + attrMetadata + "'";
-                        } else {
-                            if (attrValue == null || attrValue.equals("")) {
-                                attrValue = "NULL";
+                        if (contextAttribute.getValue().isJsonNull()) {
+                            column += ",NULL,'" + attrMetadata + "'";
+                        } else if (contextAttribute.getValue().isJsonPrimitive()) {
+                            if (contextAttribute.getValue().getAsJsonPrimitive().isBoolean()) {
+                                column += "," + attrValue.toUpperCase() + ",'"  + attrMetadata + "'";
```

Comment:
Please, add a small comment explaining why you change to uppercase.

Reply:
Fixed in [1f72cde](https://github.com/telefonicaid/fiware-cygnus/pull/1784/commits/1f72cde9337c8b9cd68d42a86307cbade02acff2)

Reply:
The same change about usage isJsonNull() will apply to https://github.com/telefonicaid/fiware-cygnus/blob/c9cd1c4c51d3a81b04790be099bb1fc5b42ed054/cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgisSink.java#L436 ?

Reply:
Not sure if I understand your question @AlvaroVega. Do you mean to replace all matches with that line?